course/02_category_widget/solution_02_category_widget/lib/main.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// You can read about packages here: https://flutter.io/using-packages/
+import 'package:flutter/material.dart';
+
+// We can also import files from relative paths
+import 'category.dart';
```

Comment:
I *highly* recommend not suggesting folks mix relative and `package` imports.  That gives our tools fits.  See for example:

https://github.com/dart-lang/sdk/issues/32042

Reply:
Thanks, can you please also review? Thanks! https://github.com/flutter/udacity-course/pull/52

Reply:
Done.  Thanks!